### PR TITLE
Skip Sentry configuration in Lando environments

### PR DESCRIPTION
**Pull request by matt-bernhardt**
Our current Sentry integration with this app results in exceptions being sent to Sentry even when working locally, which causes confusion and counts against our message quota.

This change prevents local development exceptions from being sent to Sentry. Full details are in the commit message.

### Relevant ticket(s):

https://mitlibraries.atlassian.net/browse/pw-106

### Stylesheets

- [ ] Any theme or plugin whose stylesheets have changed has had its version
      string incremented.

### Secrets

- [ ] All new secrets have been added to Pantheon tiers
- [ ] Relevant secrets have been updated in Github Actions
- [ ] All new secrets documented in README
- [x] No secrets are affected

### Documentation

- [ ] Project documentation has been updated
- [x] No documentation changes are needed

### Accessibility

- [ ] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)
- [x] There are no a11y implications to this change

### Stakeholder approval

- [ ] Stakeholder approval has been confirmed
- [x] Stakeholder approval is not needed

### Dependencies

NO dependencies are updated

## Code Reviewer

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] The changes have been verified
- [ ] The documentation has been updated or is unnecessary
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: docs/howto/troubleshoot-build-failures.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# How to troubleshoot a build failure on Pantheon
```
Comment (JPrevost) [inline docs/howto/troubleshoot-build-failures.md]: Just confirming these new docs are intended to be part of this changeset? I don't see a mention of them and they seem unrelated to the ticket.

Comment (matt-bernhardt) [reply docs/howto/troubleshoot-build-failures.md]: Oh, shoot - sorry, this is wrong. This was a document I'd started writing about rotating token values, and later morphed into the new secrets management article. I'll remove this...

Comment (matt-bernhardt) [reply docs/howto/troubleshoot-build-failures.md]: Removed...

**Review by JPrevost — APPROVED** [review]
